script.js

```diff
@@ -44,10 +44,50 @@ function logSubmit(event) {
   const newBook = new BookList();
   newBook.addBooks(bookTitle, bookAuthor);
   newBook.displayBooks();
-
+  document.querySelector('#book-title').value = '';
+  document.querySelector('#book-author').value = '';
   event.preventDefault();
 }
 
 document.querySelector('#submit-button').addEventListener('submit', (e) => logSubmit(e));
 const newBook = new BookList();
 newBook.displayBooks();
+
+// Getting date and time
+const dateTime = document.getElementById('date');
+const currentDate = new Date();
+const date = currentDate.toDateString();
+const time = currentDate.toLocaleTimeString();
+dateTime.innerHTML = `${date} ${time}`;
+
+// navigation links
+const myLinksContainer = document.getElementById('list');
+const addContainer = document.getElementById('new');
+const contactContainer = document.getElementById('contact');
+const list = document.getElementById('book-list');
+const bookForm = document.getElementById('book-form');
+const contactInfo = document.getElementById('contact-info');
+myLinksContainer.addEventListener('click', (e) => {
+  bookForm.style.display = 'none';
```

Comment:
While you did an impressive job working on the project requirements I noticed you disabled eslint in line 29. It is against microverse policy to disable linters because linters help us write clean code. So it would be nice to enable Eslint in this file.

Take a look at this screen shot:

![Screenshot 2023-05-31 at 1 56 04 PM](https://github.com/batoolfatima2135/Awesome-books/assets/79506105/9b395252-9627-4a76-9013-e8cdaa4ab005)

Please kindly fix this issue .

Reply:
hey @KayLemba, Actually we are allowed to disable eslint in a few cases. Let me show a screenshot for reference.

![Capture](https://github.com/batoolfatima2135/Awesome-books/assets/61277612/42d4f589-a4c8-4dc7-bcb5-1d229016633e)

Reply:
If you feel I was unfair in my review please kindly ask for a second opinion on this issue I follow strict guidelines when doing code reviews and in this case I havent come across that in the requirements for this project you have submitted That requirement you are showimg was allowed in your first part of the awesome books  project